buildenv/jenkins/jobs/infrastructure/Update-Reference-Repos.groovy

```diff
@@ -100,7 +98,7 @@ timeout(time: 6, unit: 'HOURS') {
                         jobs["${sNodeName}"] = {
                             node("${sNodeName}"){
                                 stage("${sNodeName} - Update Reference Repo") {
-                                    refresh(sNodeName, "${HOME}/openjdk_cache", EXTENSIONS_REPOS, true)
+                                    refresh(sNodeName, "${HOME}/openjdk_cache", VARIABLES.openj9.get('default').get('repoUrl'), true)
```

Comment:
This should be an array not a string (e.g `[[name: "openj9", url: VARIABLES.openj9.get('default').get('repoUrl')]]`), see `refresh()` signature. This should fix the failure in your build: `No such field found: field java.lang.String name` .

Reply:
That worked. I think this is ready now?

Reply:
Shouldn't be an array instead of a map?